i18n/translations/en/form-builder.json

```diff
@@ -374,14 +374,14 @@
     "somethingWentWrong": "Something went wrong",
     "downloadButtonText": "Download a copy",
     "draft": {
-      "message1": "Deleting a form cannot be undone. Your form will be removed from your forms list.",
-      "message2": "Download a copy of this form for your records. The saved JSON form file can be uploaded again to create a new form later."
+      "message1": "Deleting a form cannot be undone. Your form will be removed permanently.",
+      "message2": "Download a copy of the form file for your records. You can then upload it in GC Forms to re-create a new form later."
     },
     "okay": "Delete form",
     "published": {
-      "message1": "Deleting a form cannot be undone. Your form will be removed from your forms list and the Internet.",
-      "message2": "Download a copy of this form for your records. The saved JSON form file can be uploaded again to create a new form later.",
-      "message3": "The URL of your form will change if you upload your form again."
+      "message1": "Deleting a form cannot be undone. Your form will be removed permanently, potentially creating a broken link.",
```

Comment:
I'm curious if there's an opportunity here to point them to the close-a-form feature. Something like 

`Looking to stop submissions to your form instead? You can close your form by going to 'Form management' in 'Settings' `

--